base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -400,7 +401,7 @@ where T: BlockchainBackend
     /// Returns the transaction kernel with the given hash.
     pub fn fetch_kernel(&self, hash: HashOutput) -> Result<TransactionKernel, ChainStorageError> {
         let db = self.db_read_access()?;
-        fetch_kernel(&db, hash)
+        fetch_kernel(db.deref(), hash)
```

Comment:
curious - why not &db?

Reply:
I actually changed it back to that, but didn't push in time